ios/MarkdownTextInputDecoratorView.mm

```diff
@@ -64,18 +62,14 @@ - (void)didMoveToWindow {
   [_markdownUtils setMarkdownStyle:_markdownStyle];
 
   [_textInput setMarkdownUtils:_markdownUtils];
-  if ([_backedTextInputView isKindOfClass:[RCTUITextField class]]) {
-    RCTUITextField *textField = (RCTUITextField *)_backedTextInputView;
-    _adapter = [textField valueForKey:@"textInputDelegateAdapter"];
-    [_adapter setMarkdownUtils:_markdownUtils];
-  } else if ([_backedTextInputView isKindOfClass:[RCTUITextView class]]) {
+  if ([_backedTextInputView isKindOfClass:[RCTUITextView class]]) {
     _textView = (RCTUITextView *)_backedTextInputView;
     [_textView setMarkdownUtils:_markdownUtils];
     NSLayoutManager *layoutManager = _textView.layoutManager; // switching to TextKit 1 compatibility mode
     layoutManager.allowsNonContiguousLayout = NO; // workaround for onScroll issue
     object_setClass(layoutManager, [MarkdownLayoutManager class]);
     [layoutManager setValue:_markdownUtils forKey:@"markdownUtils"];
-  } else {
+  } else if (![_backedTextInputView isKindOfClass:[RCTUITextField class]]) {
```

Comment:
Let's revert to the original `if/else if/else` control flow and add `// Do nothing` comment in the first branch for better git blame output.